src/test/java/io/pravega/connectors/flink/FlinkTableITCase.java

```diff
@@ -195,6 +193,7 @@ public void testBatchTable() throws Exception {
                 .withRoutingKeyField("category")
                 .build();
         table.writeToSink(sink);
+        env.execute();
```

Comment:
I would like to know whether you found it necessary to call `execute` here.  I would think a single execution (line 210) would suffice.

Reply:
Yes, the data was not populated and available for the source to read from when we don't call the `execute()`.